tests/unit/serve/instrumentation/conftest.py

```diff
@@ -115,5 +115,7 @@ def _ticker(self) -> None:
                     self.collect(timeout_millis=self._export_timeout_millis)
             self.collect(timeout_millis=self._export_interval_millis)
 
+    real_reader = opentelemetry.sdk.metrics.export.PeriodicExportingMetricReader
     opentelemetry.sdk.metrics.export.PeriodicExportingMetricReader = PatchedTextReader
```

Comment:
I would change in another PR, but I believe there is a better way to use `monkeypatch` to patch for tests